### ISSUE TITLE
Hotkeys Not Highlighting

### ISSUE THREAD

**Issue by DelPyth** [maestrith/AHK-Studio#149]
Best guess is the RegEx for hotkeys is off, or something. As far as I know, it should have a highlight color but nothing's being colored.
- 11/23/2017 10:19 PM

**Comment by derz00** [comment]
Best guess is Studio doesn't use RegEx for highlighting?
Any screenshots? What version of Studio?
👋

**Comment by maestrith** [comment]
@DelPyth (\ not too sure what you mean

**Comment by DelPyth** [comment]
@maestrith /)
Sorry for my lack of info
Here's a little screen shot
Keep in mind, I'm using the latest build I can.
![capture - 24 11](https://user-images.githubusercontent.com/23638456/33215972-d7b9f104-d0ff-11e7-9c69-f0864ab8528b.png)

**Comment by maestrith** [comment]
AH! ok.  Hotkeys that are just KeyName:: are fine, just not the ones with any sort of !^...etc

**Comment by DelPyth** [comment]
Yes, hotstrings don't seem to be highlighted as hotkeys either. It's quiet interesting.

**Comment by maestrith** [comment]
There is a problem.  Hotkeys can take on just about any form... Any ideas as to how to detect them? They can have any key on the keyboard, spaces, nearly any symbol.... It is a mess.
